web/themes/new_weather_theme/templates/block/block--weathergov-location-search.html.twig

```diff
@@ -1,7 +1,9 @@
 <div>
 
   {% if content.place %}
-  <h1 class="margin-bottom-0">{{ content.place.city }}, {{ content.place.state }}</h1>
+  <h1 class="margin-bottom-0">
+    {{ content.place.city }}{% if content.place.state %}, {{ content.place.state }}{% endif %}
+  </h1>
```

Comment:
This is because the `place.state` property may not always be set anymore. For example, if the user selects "Sea World, 500 Sea World Dr, San Diego, CA, 92109, USA", we won't parse that into city/state. We'll just stuff everything into the `city` property. In that case, we don't want the extraneous comma.